app/helpers/tags_helper.rb

```diff
@@ -76,7 +76,11 @@ def link_to_tag_works(tag, options = {})
   end
 
   def link_to_tag_with_text(tag, link_text, options = {})
-    link_to_with_tag_class(@collection ? collection_tag_path(@collection, tag) : tag_path(tag), link_text, options)
+    if options[:full_path] 
```

Comment:
Would it be possible to do something like this to make it DRYer?
```
def link_to_tag_with_text(tag, link_text, options = {})
  collection_url_for_tag = options[:full_path] ? collection_tag_url(@collection, tag) : collection_tag_path(@collection, tag)
  link_to_with_tag_class(@collection ? collection_link_for_tag : tag_path(tag), link_text, options)
end
```

My other question is do we _definitely_ always want `tag_path`, or should it be `tag_url` when `options[:full_path]`?